spec/classes/datadog_agent_integrations_docker_daemon_spec.rb

```diff
@@ -6,17 +6,18 @@
     return
   end
 
-  context 'supported agents' do
-    ALL_SUPPORTED_AGENTS.each do |agent_major_version|
+  ALL_SUPPORTED_AGENTS.each do |agent_major_version|
+    context 'supported agents' do
       let(:pre_condition) { "class {'::datadog_agent': agent_major_version => #{agent_major_version}}" }
 
-      if agent_major_version == 5
-        let(:conf_file) { '/etc/dd-agent/conf.d/docker.yaml' }
-      else
-        let(:conf_file) { "#{CONF_DIR}/docker.d/conf.yaml" }
-      end
+      conf_file = if agent_major_version == 5
+                    '/etc/dd-agent/conf.d/docker_daemon.yaml'
```

Comment:
Out of curiosity, do you know why is this one name is different (`docker_daemon` v `docker`)?

Reply:
It matches the code this way: https://github.com/DataDog/puppet-datadog-agent/blob/master/manifests/integrations/docker_daemon.pp#L100

It might be that this got changed at some point for A6.